src/storage-types/src/sinks.rs

```diff
@@ -279,23 +278,6 @@ impl RustType<ProtoSinkAsOf> for SinkAsOf<mz_repr::Timestamp> {
     }
 }
 
-impl RustType<proto::SinkAsOf> for SinkAsOf<mz_repr::Timestamp> {
-    fn into_proto(&self) -> proto::SinkAsOf {
-        proto::SinkAsOf {
-            frontier: Some(self.frontier.into_proto()),
-            strict: self.strict,
-        }
-    }
-
-    fn from_proto(proto: proto::SinkAsOf) -> Result<Self, TryFromProtoError> {
-        let frontier = proto.frontier.into_rust_if_some("SinkAsOf::frontier")?;
-        Ok(SinkAsOf {
-            frontier,
-            strict: proto.strict,
-        })
-    }
-}
```

Comment:
FYI to storage folks, if you look above this, there's an almost identical protobuf object called `ProtoSinkAsOf`. I noticed that for pretty much all objects used in the storage controller stash, there were duplicate protobuf objects.